docs/tutorials/plots.rst

```diff
@@ -1057,3 +1059,39 @@ make multiple plots:
     plt.show()
 
 :ref:`Return to Top <plots>`
+
+.. _finder_image:
+
+Finder Chart/Image
+==================
+
+`astroplan` includes a function for generating quick finder images from the
+command line, `~astroplan.plots.plot_finder_image`, by querying for images from
+sky surveys centered on a `~astroplan.FixedTarget`. This function depends on
+`astroquery <https://astroquery.readthedocs.org/en/latest/>`_  (in addition
+to `Matplotlib`_). In this example, we'll quickly make a finder image centered
+on The Crab Nebula (M1):
+
+.. code-block:: python
```

Comment:
Is it necessary / useful to duplicate this code example?
Can't you add `:include-source:` to the `.. plot::` directive and avoid the code duplication that way?